autoflake.py

```diff
@@ -1192,6 +1206,11 @@ def _main(argv, standard_out, standard_error, standard_input=None):
         '--remove-unused-variables', action='store_true',
         help='remove unused variables',
     )
+    parser.add_argument(
+        '--remove-rhs-for-unused-variables', action='store_true',
```

Comment:
Can you also add `remove-rhs-for-unused-variables` to the config function? (The validation in the function `merge_configuration_file`)

Reply:
done